pkg/rancher-desktop/backend/containerClient/mobyClient.ts

```diff
@@ -65,7 +65,8 @@ export class MobyClient implements ContainerEngineClient {
   }
 
   async waitForReady(): Promise<void> {
-    let successCount = 0;
+    let successCount = 0; let failureCount = 0;
```

Comment:
I have a slight preference for only using one statement per line:

```suggestion
    let successCount = 0;
    let failureCount = 0;
```

Reply:
Huh, I had `successCount = 0, failureCount = 0`, I guess the linter fixed it to use two statements but somehow didn't make it into separate lines.

If it's two statements then yeah I think one per line is better too. Will fix.